<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoverPathDialog</class>
 <widget class="QDialog" name="CoverPathDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>覆盖式轨迹属性设定</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>130</width>
     <height>80</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_coverPath_pre">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>105</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>预计覆盖区域</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPathWidth">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;轨迹宽度=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="pathWidthSpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>65</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="suffix">
    <string>cm</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>140</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelMinTurnRadius">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;转弯半径=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="minTurnRaiusSpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>105</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="suffix">
    <string>m</string>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelPathType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;航向规划=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPathType">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>145</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>靠右</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelStartPointF">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;起始位置=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelStartPointF_data">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>20</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;（x，y）&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectPoint">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>25</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>选点</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInitDirection">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;初始方向=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBox_initDirection">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>185</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="suffix">
    <string>°</string>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>45.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
